clients/packages/sdk/src/client/apis/DocumentedApi.ts

```diff
@@ -0,0 +1,254 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * Polar API
+ * Read the docs at https://docs.polar.sh/api
+ *
+ * The version of the OpenAPI document: 0.1.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+import * as runtime from '../runtime';
+import type {
+  HTTPValidationError,
+  LicenseKeyActivationRead,
+  LicenseKeyRead,
+  LicenseKeyUpdate,
+  LicenseKeyWithActivations,
+  ListResourceLicenseKeyRead,
+  ResourceNotFound,
+  Unauthorized,
+} from '../models/index';
+
+export interface DocumentedApiGetRequest {
+    id: string;
+}
+
+export interface DocumentedApiGetActivationRequest {
+    id: string;
+    activationId: string;
+}
+
+export interface DocumentedApiListRequest {
+    organizationId: string;
+    page?: number;
+    limit?: number;
+}
+
+export interface DocumentedApiUpdateRequest {
+    id: string;
+    body: LicenseKeyUpdate;
+}
+
+/**
+ * 
+ */
+export class DocumentedApi extends runtime.BaseAPI {
```

Comment:
🚨 The name is not good, it should be `LicenseKeys`. Add a tag on the router to fix this (see previous comment).